docs/UserGuide.md

```diff
@@ -145,22 +145,30 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
 
 * Task name should be in a pair of quotation marks.
 
-* The floating task will be stored and shown to user according to the order of
-  the time they created.
+* Task name could be a single word or a phrase, white space are allowed.
+
+* The floating task will be shown according to their `PRIORITY`.
+
+* Two floating tasks with the same `PRIORITY` will be shown according to
+  the order of the time they created.
 
 * The `PRIORITY` attribute is an integer which ranges from `0` to `5`.
 
 * The default piority of the floating task is 0.
```

Comment:
We'll continue the discussion in https://github.com/CS2103AUG2016-T11-C4/main/pull/39#discussion_r82523900